lib/shared/size_config.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:flutter/material.dart';
+
+class SizeConfig {
+  static MediaQueryData _mediaQueryData;
+  static double screenWidth;
+  static double screenHeight;
+  static double blockSizeHorizontal;
+  static double blockSizeVertical;
+
+  static double _safeAreaHorizontal;
+  static double _safeAreaVertical;
+  static double safeBlockHorizontal;
+  static double safeBlockVertical;
```

Comment:
Am I really going to use all of these?  Should I make one for other circumstances?  E.g. text sizing

Reply:
I should not need one for text sizing, since that would just be based on the relative height/width anyway

Refactored and done